### PR TITLE
Add `max_request_body_size` option

### PR DESCRIPTION
**Pull request by mpeeters**
related to issue #155

### CODE REVIEW

**Comment by mpeeters** [comment]
@jenkins-plone-org please run jobs

File: src/plone/recipe/zope2instance/recipe.py [hunk 1]
```diff
@@ -815,6 +815,7 @@ def build_wsgi_ini(self):
             'listen_port': '8080',
             # /todo
             'location': options['location'],
+            'max_request_body_size': options.get('max_request_body_size', '1073741824'),
```
Comment (icemac) [inline src/plone/recipe/zope2instance/recipe.py:818]:
```suggestion
            'max_request_body_size': options.get(
                'max_request_body_size', '1073741824'),
```

This line seems to be too long according to the flake8 check.

**Review by icemac — APPROVED** [review]
LGTM besides the broken flake8 check.

**Review by mauritsvanrees — APPROVED** [review]
LGTM, except the flake8 error, as Michael said.

Well, technically the default should not be a string, but an integer, so it would be nice to change that.
But in practice it will be written without quotes to the `wsgi.ini`.

**Comment by mauritsvanrees** [comment]
Actually, would you consider changing the option to `max-request-body-size` with dashes? That is what all other options use, except for a few sentry options.
The resulting `wsgi.ini` file should still contain `max_request_body_size` of course.

**Comment by mpeeters** [comment]
@mauritsvanrees @icemac I will do the change for flake 8 and use dashes instead of underscores.

**Comment by mauritsvanrees** [comment]
Excellent, thanks. I will merge.

**Pull request merged**